bundles/org.openhab.binding.clementineremote/src/main/resources/OH-INF/i18n/clementineremote_de.properties

```diff
@@ -0,0 +1,34 @@
+# binding
+
```

Comment:
With the initial contribution, only the default (english) should be provided. All additional languages should be contributed by crowdin. Please remove this file

Reply:
Damn. This re-emerged when cherry-picking from the 3.3.0 branch I'm developing on. Removed in the latest commit.